tests/run.bats

```diff
@@ -288,6 +298,18 @@ function configure_and_check_user() {
   expect_output "888:888"
 }
 
+@test "run --workingdir" {
+	skip_if_no_runtime
+
+	_prefetch alpine
+	run_buildah from --quiet --pull=false --signature-policy ${TESTSDIR}/policy.json alpine
+	cid=$output
+	run_buildah run $cid pwd
+	expect_output "/"
+	run_buildah run --workingdir /bin $cid pwd
+	expect_output "/bin"
+}
```

Comment:
Could you add a separate run command

run_buildah run $cid pwd
expect_ouput != "/bin" (Not sure how to do this without reading the tests.